README.md

```diff
@@ -41,7 +46,7 @@ Using Opam, a suitable OCaml environment can be setup as follows.
 ```bash
 opam switch 4.05.0
 eval `opam config env`
-opam install dune odoc menhir yojson cmdliner bindlib.5.0.0 timed.1.0 earley.2.0.0 ppx_inline_test why3.1.2.0
+opam install dune odoc menhir yojson cmdliner bindlib.5.0.0 timed.1.0 earley.2.0.0 why3.1.2.0
```

Comment:
no need to give version numbers

Reply:
No, version numbers are important.

Reply:
It may not compile otherwise.

Reply:
But that's the job of opam to figure out the version numbers so that it compiles!